scripts/generate_client_publish_job.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import os, subprocess
+import yaml
+
+WORKSPACE = os.environ.get("WORKSPACE")
+if WORKSPACE is None:
+    raise RuntimeError("provide the WORKSPACE variable!")
+
+
+def generate(trigger_from_repo, version_to_publish="master"):
+    release_tool = os.path.join(WORKSPACE, "integration", "extra", "release_tool.py")
+    integration_versions = subprocess.run(
```

Comment:
I suggest we use `check_output` (or `run(... check=True)` for all that matters) so that the call raises an error on `release_tool` errors.

Reply:
👌 added - except for the one with the piped `cut` call (as I was not not sure what the cut was supposed to cut, as my own attempts of the prior `release_tool` returned nothing to cut really)...